.travis.yml

```diff
@@ -11,6 +11,8 @@ addons:
     packages:
     - icnsutils
     - graphicsmagick
+before_script:
+- export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start || true
```

Comment:
Hmm, there was a reason for the `|| true`. Should've left a comment, my bad. I'll go poke around and see if I can find out again.